sonaranalyzer-dotnet/src/SonarAnalyzer.RuleDescriptorGenerator/Descriptor.cs.nuspec

```diff
@@ -17,7 +17,7 @@
   </metadata>
   <files>
     <!-- If these are changed than the build\release.ps1 should also be changed -->
```

Comment:
is this comment still actual? I don't see any `release.ps` in the changelist

Reply:
I think we can remove the comment. We don't have a `release.ps1` file in the project.